s3_scrubber/src/main.rs

```diff
@@ -111,13 +69,24 @@ async fn tidy(
     depth: TraversingDepth,
     skip_validation: bool,
 ) -> anyhow::Result<()> {
-    let binary_name = env::args()
-        .next()
-        .context("binary name in not the first argument")?;
-
     let dry_run = !cli.delete;
-    let _guard = init_logging(&binary_name, dry_run, node_kind.as_str());
-    let _main_span = info_span!("tidy", binary = %binary_name, %dry_run).entered();
+    let file_name = if dry_run {
+        format!(
+            "{}_{}_{}__dry.log",
+            CLI_NAME,
+            node_kind,
+            chrono::Utc::now().format("%Y_%m_%d__%H_%M_%S")
```

Comment:
personally I'd like something integrating the ISO 8601 `YYYY-MM-DD` (and `CLI_NAME` already includes a `-`), but this is just a minor point (also you are just moving the code).

Reply:
This is existing code that got moved.  I'm personally not very attached to the way this tool writes out a unique log each run, but let's save changing that for another PR.